CHANGELOG.md

```diff
@@ -9,7 +9,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Added `table_columns` parameter to `civis.io.civis_file_to_table`, `civis.io.dataframe_to_civis`, and `civis.io.csv_to_civis` (#379)
 
 ### Fixed
-
+- No longer require ServiceClient to be instantiated to parse a
```

Comment:
I think this should go under "Changed".

Also, I think this is technically a breaking change, unless we want to keep the `parse_path` and `parse_api_spec` functions and have them call the new ones.  I think it'd be a good idea to replace `## Unreleased` here with `## 2.0.0 (Unreleased)` here as a reminder if we're going to keep the breaking change.

Reply:
Oops, I thought I moved it to "changed", my mistake.

I can add the note for 2.0.0. Would making this breaking change impact when you'll want to cut a release? I know there's some other changes throughout the repo that are set for the next major version.

As an alternative, I could follow your suggestion to make it a non-breaking change, and add a deprecation tag that the duplicate methods will be dropped in 2.0.0.

Reply:
If you think it wouldn't be too bad to make some pass-through calls with deprecation warnings, that seems better.

Regarding the "will be removed in v2.0.0" warnings, I think we could just update those to say "v3.0.0" or "a future release" to avoid delaying release.  Same for the 2.0 milestone in github.